resources/config.rb

```diff
@@ -0,0 +1,73 @@
+property :name, String, name_property: true
+property :home_net, String,     default: 'any'
+property :external_net, String, default: 'any'
```

Comment:
yeah it's pretty horrible. I'm really only using it to get *some* config on disk. 

I'm going to recommend people put their own files on disk.